pkg/subctl/cmd/verify.go

```diff
@@ -0,0 +1,104 @@
+/*
+© 2020 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/onsi/ginkgo/config"
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/shipyard/test/e2e/framework"
+
+	verifyconnectivity "github.com/submariner-io/submariner-operator/pkg/subctl/verify/connectivity"
+	verifyservicediscovery "github.com/submariner-io/submariner-operator/pkg/subctl/verify/servicediscovery"
+)
+
+var (
+	verifyConnectivity     bool
+	verifyServiceDiscovery bool
+	verifyAll              bool
+	//verboseConnectivityVerification bool
+	//operationTimeout                uint
+	//connectionTimeout               uint
+	//connectionAttempts              uint
+	//reportDirectory                 string
+)
+
+func init() {
+	verifyCmd.Flags().BoolVar(&verifyConnectivity, "connectivity", true, "verify connectivity between two clusters")
+	verifyCmd.Flags().BoolVar(&verifyServiceDiscovery, "service-discovery", false, "verify service-discovery between two clusters")
+	verifyCmd.Flags().BoolVar(&verifyAll, "all", false, "verify connectivity and service-discovery between two clusters")
+	verifyCmd.Flags().BoolVar(&verboseConnectivityVerification, "verbose", false, "Produce verbose logs during connectivity verification")
```

Comment:
Let's move the common flags to a function:

https://github.com/submariner-io/submariner-operator/blob/master/pkg/subctl/cmd/join.go#L77

like we do in the subctl/cmd/broker.go

Reply:
Will do.